src/server/generic_family.cc

```diff
@@ -195,9 +195,8 @@ std::optional<DbSlice::ItAndUpdater> RdbRestoreValue::Add(std::string_view data,
     return std::nullopt;
   }
 
-  auto res = db_slice.AddNew(cntx, key, std::move(pv), args.ExpirationTime());
-  res->it->first.SetSticky(args.Sticky());
-  if (res) {
+  if (auto res = db_slice.AddNew(cntx, key, std::move(pv), args.ExpirationTime()); res) {
```

Comment:
This makes sense. My question here is why do Migrations use `RdbRestoreValue::Add` ?

Reply:
I don't know, it's according design to use RESTORE command